stance/predict/overlay.py

```diff
@@ -13,7 +16,51 @@ def display_overlay(input_video, template_skeletons):
         path to the video of the user performing the motion
     template_skeletons : List[Skeletons]
     """
-    pass
+    cap = cv2.VideoCapture(input_video)
+    numFrames = int(cap.get(cv2.CAP_PROP_FRAME_COUNT))
```

Comment:
Instead of this, consider using a `while success` the same way @aspadiyath  did in `etl.py`

Reply:
I second this